mm/2s2h/Enhancements/Audio/AudioEditor.cpp

```diff
@@ -416,17 +436,25 @@ void DrawTypeChip(SeqType type) {
     ImGui::EndDisabled();
 }
 
-void AudioEditorRegisterOnSceneInitHook() {
-    // BENTODO implement this
-    // GameInteractor::Instance->RegisterGameHook<GameInteractor::OnSceneInit>([](int16_t sceneNum) {
-    //    if (CVarGetInteger(CVAR_AUDIO("RandomizeAllOnNewScene"), 0)) {
-    //        AudioEditor_RandomizeAll();
-    //    }
-    //});
+void AudioEditorRegisterOnSceneChangeHook() {
+    GameInteractor::Instance->RegisterGameHook<GameInteractor::OnSceneChange>([](int16_t sceneNum) {
+        if (CVarGetInteger("gAudioEditor.RandomizeAllOnNewScene", 0)) {
```

Comment:
The pattern in this file is to use `CVAR_AUDIO(var)`, so for consistency's sake, let's change these CVar calls.


```suggestion
        if (CVarGetInteger(CVAR_AUDIO("RandomizeAllOnNewScene"), 0)) {
```

I realize this may be moot when it comes to addressing CVar prefixes in general, but within the scope of this PR, I'd prefer we not have "gAudioEditor.Something" 10 times in one file.